src/cpp/server/DBActiveSessionStorage.cpp

```diff
@@ -0,0 +1,436 @@
+/*
+ * DBActiveSessionStorage.cpp
+ *
+ * Copyright (C) 2022 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+
+#include "server/DBActiveSessionStorage.hpp"
+#include "server_core/ServerDatabase.hpp"
+#include "core/Database.hpp"
+#include <numeric>
+
+namespace rstudio
+{
+namespace server
+{
+namespace storage
+{
```

Comment:
For namespace blocks we prefer to keep the opening brace on the same line

```suggestion
namespace rstudio {
namespace server {
namespace storage {
```